hardhat.config.ts

```diff
@@ -6,6 +6,7 @@ import "@nomiclabs/hardhat-waffle";
 import "@typechain/hardhat";
 import "hardhat-gas-reporter";
 import "solidity-coverage";
+import "hardhat-deploy";
```

Comment:
To use fixtures, hardhat deploy was needed.

Reply:
welcome back my old friend